tutorials/marble/version-control.md

```diff
@@ -0,0 +1,393 @@
+
+![Create Branch Successful](images/version-control/create-branch-successful.png)
+
+### Stashing Files
+
+Stashing a file means to temporarily store files you're working on so the changes made to them will not affect other files, 
+and changes to other files will not affect the working file.  Stashing is typically done before switching branches, so 
+the changes from other branches will not overwrite changes in files you are currently working on.
+
+Currently there is no mechanism in the GUI to stash files.  Files can only be stashed from the Terminal.
+
+[Stash Files from Terminal](#stash-terminal)
+
+Once the command is run all files listed under `Changed` will be moved to the `Stash` and will not be seen. 
+
+
+### Switching Branches
+
+```{note}
+You can only switch branches when you have no files staged or changed.
+ 
+To switch branches simply click another branch listed under the `Branches` section.
+```
+
+```{note}
+You can only switch to another local branch.  Branches with the `origin` prefix are remote branches and cannot be switched into.
+``` 
+
+
+### Using Git with the Terminal
+
+### Set Up Your Git Identifier
+
+If you are working in a team this will show who made a certain commit.
+
+```
+git config --global user.name "Your Name"
+git config --global user.email "email@email.com"
+```
+
+### Staging a File
+
+Make sure you are in the same directory as the file you want to stage.
+
+```
+git add <filename>
+```
+
+### View Staged Files
+
+If you have multiple files staged and ready to commit you can see the list by using `git status`.
+
+```
+git status
+```
+
+
+### Making a Commit
+
+Add a brief descriptive message about your commit.
+
+```
+git commit -m "your brief commit message goes here"
+```
+
+### Pushing a Commit
+
+After your commit is complete, push it to the remote repository
+
+```
+git push
+```
+
+### Stashing Files
+
+Make sure you are in the same directory as the file you want to stash.  
+
+Running the `stash` command will stash all files that have changes made to them.
+
+```
+git stash
+```
+
+To see the files you stashed you can list them: 
+
+```
+git stash list
+```
+
+To resume working on your stashed files you will need to get them out of the stash.  Run the following command and you will see them under the `Changed` section.
+
+```
+git stash pop
+```
+
+
+### Creating a Branch
+
+```
+git branch <new-branch-name>
+```
+
+### Switching Branches
+
+```{note}
+You can only switch branches when you have no files staged or changed.
+``` 
+
+Switch branches by checking out the branch.  Replace *<branch-name>* with the branch you want to switch to.
+
+```
+git checkout <branch-name>
+```
+
+```{note}
+You can only switch to another local branch.  Branches with the `origin` prefix are remote branches and cannot be switched into.
+``` 
+
+## Creating a Pull Request
```

Comment:
Move this into a separate section about GitHub specifically

Reply:
Moved Pull Request section under GitHub section